x-pack/plugins/enterprise_search/public/applications/workplace_search/routes.ts

```diff
@@ -17,34 +17,36 @@ export const LOGOUT_ROUTE = '/logout';
 export const LEAVE_FEEDBACK_EMAIL = 'support@elastic.co';
 export const LEAVE_FEEDBACK_URL = `mailto:${LEAVE_FEEDBACK_EMAIL}?Subject=Elastic%20Workplace%20Search%20Feedback`;
 
-export const DOCS_PREFIX = docLinks.workplaceSearchBase;
-export const DOCUMENT_PERMISSIONS_DOCS_URL = docLinks.workplaceSearchDocumentPermissions;
-export const PRIVATE_SOURCES_DOCS_URL = docLinks.workplaceSearchPermissions;
-export const EXTERNAL_IDENTITIES_DOCS_URL = docLinks.workplaceSearchExternalIdentities;
-export const SECURITY_DOCS_URL = docLinks.workplaceSearchSecurity;
 export const BOX_DOCS_URL = docLinks.workplaceSearchBox;
 export const CONFLUENCE_DOCS_URL = docLinks.workplaceSearchConfluenceCloud;
 export const CONFLUENCE_SERVER_DOCS_URL = docLinks.workplaceSearchConfluenceServer;
+export const CUSTOM_SOURCE_DOCS_URL = docLinks.workplaceSearchCustomSources;
```

Comment:
I question whether we need these at all now. Since all of these constants merely point to `docsLinks` properties, shouldn't we just import `docsLinks` directly into the components so we don't have to maintain these references and this file can be used for internal app routes only? I understand that this adds a bit of work to this PR, but I believe it's the best solution, long-term. 

Thoughts?

Reply:
Yes, that's definitely the long-term solution used in other apps. I'm just doing the stop-gap method to have the least impact across the app.

Reply:
Thanks!

Reply:
Issue created https://github.com/elastic/workplace-search-team/issues/2191